### DISCUSSION TITLE
Hacktoberfest-like event dedicated to InnerSource

### DISCUSSION THREAD

**Discussion by spier** [InnerSourceCommons/InnerSourceMarketing#246] [Ideas]
Completely random and raw idea, so don't kill the baby immediately :slightly_smiling_face:

Every October DigitalOcean organizes [hacktoberfest](https://hacktoberfest.com/).

What is it, and why does DigitalOcean organize it? From their website:

> Hacktoberfest is DigitalOcean’s annual event that encourages people to contribute to open source throughout October.
> 
> Much of modern tech infrastructure—including some of DigitalOcean’s own products—relies on open-source projects built and maintained by passionate people who often don’t have the staff or budgets to do much more than keep the project alive. 
> 
> Hacktoberfest is all about 
> * **giving back to those projects**, 
> * **sharpening skills**, and 
> * **celebrating all things open source**, especially the people that make open source so special.

The ISC can participate in hacktoberfest as well, and try to get open source contributions for our ISC resources (patterns, learning path, etc).
We have done that in the past, at least a little bit. However from what I can tell with only little effect.

## Toying with another Idea

Could we as the ISC organize an event that somehow encourages and celebrates InnerSource contributions?
Obviously harder, as you cannot see them in the "open" :slightly_smiling_face:

Maybe the event could run in October as well, to create a venue for people that cannot participate in open source, for one reason or the other.

Need to chew on this more to see if we can come up with something that we are really excited about.
Already wanted to share it though to see if anybody could help to make this work.

**Comment by claredillon** [comment]
On the celebrating InnerSource contributions / contributors... another idea: maybe we could have this separate to Hacktober - which is very associated with open source... (and we might risk overloading that event, and if we tried to celebrate non-OSS contributions in the same event, some folks might not like that). So maybe we could make an actual InnerSource day (and logistically put in a bit further away from the Summit so more folks could focus on helping). 

In the past, the developer evangelism groups in Microsoft used to have the idea of "an event in a box" which was a set of resources (design/content resources) that any local team could take to run an event - all happening around the same time. So, the effort of organizing was distributed, but everyone felt like they were part of something bigger...  

An associated idea would be to have InnerSource award day, where folks in companies nominate their top InnerSource contributors for a ISC award?

**Reply by rrrutledge** [reply]
We could create event-in-a-box and have people from the ISC sign up to run those in their companies.  Then we could do social website posting list the winners from each company.

**Reply by spier** [reply]
> To get access to the event-in-a-box resources, please join our slack channel ...

To help us bring new people into our community :)

**Comment by claredillon** [comment]
And building on the previous conversation about travel funds drying up - if it were available to local community groups as well as folks wanting to do it in their company - it could really extend awareness of InnerSource!

**Comment by spier** [comment]
What would be the merits of an Hacktoberfest-like event organized by the ISC? [^1]

Brainstorming a few:

* the ISC establishing itself as a driver of the InnerSource movement
* the ISC giving local organizers in a company an easier path to organize an event. as in "look, all these other orgs are participating in the event too"
* the ISC reducing the amount of time that an org needs to invest themselves. they should be able to participate at the event with minimal time investment  
* discovering new organizations that are already practicing InnerSource (by hearing the stories that they share during the event) - possibly filling the [stories page](https://innersourcecommons.org/stories/) with more orgs
* recruiting new contributors for the ISC
* allowing orgs to "launch their InnerSource initiative" (not sure if such an event could lend itself for this actually)
* allowing orgs to test what the level of interest in InnerSource within their orgs is
* to be continued

[^1]: Note that the level of "organization" the ISC really does is for us to define. e.g. above others are discussing the event-in-a-box concept, where the ISC would provide a pre-packaged concept for an event, that then needs to be run/executed by somebody local in an org.

**Reply by claredillon** [reply]
don't want to lose this conversation either Spier - which I recognise is separate to the event in a box. Just to be clear - I'm assuming the goal here is for OSS contributions to be to ISC in this case (vs. celebrating InnerSource contributions)

**Comment by rrrutledge** [comment]
“March into InnerSource”?

**Reply by claredillon** [reply]
Make May = "May the InnerSource Be With You" Month?
